test/django/test_email_backend.py

```diff
@@ -0,0 +1,190 @@
+from sparkpost.django.email_backend import SparkPostEmailBackend
+from sparkpost.django.exceptions import UnSupportedParam, UnSupportedContent
+from sparkpost.transmissions import Transmissions
+from django.conf import settings
+from django.core.mail import send_mail, send_mass_mail, EmailMessage
+import pytest
+import mock
+
```

Comment:
for imports let's follow this convention:

- stdlib imports first, in alphabetical order, newline after
- library imports (e.g. pytest, django, mock) next, in alphabetical order, newline after
- project-specific imports (e.g. sparkpost.something) next, in alphabetical order